### PR TITLE
[CLI] SDRS: Active domain and job implementation

### PR DESCRIPTION
**Pull request by artem-lifshits**
- active domain list
- job show
- tests
- documentation

### CODE REVIEW

**Review by anton-sidelnikov — DISMISSED** [review]

**Review by irina-pereiaslavskaia — DISMISSED** [review]

File: otcextensions/tests/unit/osclient/sdrs/v1/test_active_domains.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        cmp_data = (
+            obj['domains'][i].id,
+            obj['domains'][i].name,
+            obj['domains'][i].description,
+            obj['domains'][i].sold_out,
+            obj['domains'][i].local_replication_cluster.availability_zone,
+            obj['domains'][i].remote_replication_cluster.availability_zone
+        )
+
+        self.assertEqual(data, cmp_data)
```
Comment (Polina-Gubina) [inline otcextensions/tests/unit/osclient/sdrs/v1/test_active_domains.py:63]: please, add test_default

File: otcextensions/tests/unit/osclient/sdrs/v1/test_job.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+        cmp_data = (
+            obj.job_id,
+            obj.status,
+            obj.job_type,
+            obj.begin_time,
+            obj.end_time
+        )
+
+        self.assertEqual(data, cmp_data)
```
Comment (Polina-Gubina) [inline otcextensions/tests/unit/osclient/sdrs/v1/test_job.py:68]: same

File: otcextensions/tests/unit/osclient/sdrs/v1/test_active_domains.py (hunk 1 above)
Comment (artem-lifshits) [reply otcextensions/tests/unit/osclient/sdrs/v1/test_active_domains.py:63]: added test_default

File: otcextensions/tests/unit/osclient/sdrs/v1/test_job.py (hunk 2 above)
Comment (artem-lifshits) [reply otcextensions/tests/unit/osclient/sdrs/v1/test_job.py:68]: fixed

**Review by anton-sidelnikov — DISMISSED** [review]

File: otcextensions/tests/unit/osclient/sdrs/v1/test_job.py [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+    def test_default(self):
+        arglist = [
+            'job'
+        ]
+
+        verifylist = []
```
Comment (Polina-Gubina) [inline otcextensions/tests/unit/osclient/sdrs/v1/test_job.py]: maybe verifylist=('job', 'job')?

**Review by Polina-Gubina — DISMISSED** [review]

File: otcextensions/tests/unit/osclient/sdrs/v1/test_job.py (hunk 3 above)
Comment (artem-lifshits) [reply otcextensions/tests/unit/osclient/sdrs/v1/test_job.py]: done

**Review by irina-pereiaslavskaia — DISMISSED** [review]

**Review by anton-sidelnikov — DISMISSED** [review]

**Review by anton-sidelnikov — APPROVED** [review]

**Pull request merged**